.travis.yml

```diff
@@ -52,18 +52,33 @@ matrix:
       env:
         - DEPS=latest
 
+before_install:
+  - shopt -s expand_aliases
+  - alias composer="travis_retry docker-compose run --rm php composer"
+  - alias php="travis_retry docker-compose run -e TRAVIS -e TRAVIS_JOB_ID --rm php php"
+  - docker-compose build --build-arg PHP_VERSION=${TRAVIS_PHP_VERSION} --build-arg XDEBUG=${TEST_COVERAGE:+1} --no-cache php
+  - composer --version
+  - php -v
+  - php -m
+
 install:
+  - php --info
   - composer install $COMPOSER_ARGS --ignore-platform-reqs
+  - if [[ $LEGACY_DEPS != '' ]]; then composer update $COMPOSER_ARGS --with-dependencies $LEGACY_DEPS ; fi
```

Comment:
This variable is never defined for any jobs, is it?

Reply:
ADAPTER_DEPS was used to toggle testing between 5.x and 7.x.  The alcaeus/mongo-php-adapter iis now included in composer.json.

Reply:
Removed